docs/src/documentation/05-development/04-migration-guides/01-v20.mdx

```diff
@@ -12,6 +12,52 @@ With this guide, we aim to walk through all the breaking changes and how they ca
 
 ## Breaking changes
 
+### Badge component removed the `border` prop
+
+The `Badge` component removed the `border` prop, as it was not being used internally.
+
+Please adapt your code to remove the `border` prop. No visual changes are expected.
+
+**Before:**
+
+```jsx
+import Badge from "@kiwicom/orbit-components/lib/Badge";
+
+<Badge border={false} />;
+```
+
+**Now:**
+
+```jsx
+import Badge from "@kiwicom/orbit-components/lib/Badge";
+
+<Badge />;
+```
+
+### Modal component removed `autoFocus` prop
+
+The `Modal` component removed the `autoFocus` prop, as the focus management is now handled automatically to comply with accessibility.
+
+Therefore, this prop has no effect now. The first focusable element within the modal will be focused automatically when the modal is opened.
+
+This prop had a default value of `true`, therefore, the default behavior remains the same.
+
+**Before:**
+
+```jsx
+import Modal from "@kiwicom/orbit-components/lib/Modal";
+
+<Modal autoFocus={false} />;
+```
+
+**Now:**
+
+```jsx
+import Modal from "@kiwicom/orbit-components/lib/Modal";
+
+<Badge />;
```

Comment:
```suggestion
<Modal />;
```

Reply:
Wow, the fact that I **did not** copy-paste makes me wonder if the rebase messed up 🤯 Well spotted, thanks